src/explorer/explore.jl

```diff
@@ -65,3 +67,12 @@ function explore(m::Model, comp_name::Symbol, datum_name::Symbol)
 
     return VegaLite.VLSpec{:plot}(spec)
 end
+
+"""
+    function vega_plot(filepath::String, p::VegaLite.VLSpec{:plot})
+
+Save a the plot `p` as `filepath`.  Simple wrapper function for VegaLite's `save` function.
+"""
+function save_vega_plot(filepath::String, p::VegaLite.VLSpec{:plot})
+    VegaLite.save(filepath, p);
+end
```

Comment:
I think we can remove this whole function before we merge, right?

Reply:
Yes, thought I did that, thanks!